tests/benchdnn/graph/graph.hpp

```diff
@@ -42,7 +42,8 @@ struct settings_t : public base_settings_t {
 
     // ctor to save certain fields from resetting
     settings_t(const char *perf_template) : settings_t() {
-        this->perf_template = perf_template;
+        this->perf_template
+                = perf_template; // NOLINT(cppcoreguidelines-prefer-member-initializer)
```

Comment:
I think it should be fine to initialize it in the list after `settings_t()` gets created. At least I'd give it a try.

Reply:
Yes, I tried it before deciding to suppress the warning. It will not compile if we have both member initializer and the constructor delegation (`settings_t()`). I learnt from a [stackoverflow post](https://stackoverflow.com/questions/12190051/member-initialization-while-using-delegated-constructor).